src/components/video-embed/video-embed.module.css

```diff
@@ -0,0 +1,15 @@
+.playerWrapper {
+  margin-bottom: 24px;
+  padding-top: 56.25%; /* Player ratio: 100 / (1280 / 720) */
+  position: relative;
+}
+
+.reactPlayer {
+  background-color: white;
+  border-radius: 6px;
+  box-shadow: var(--token-surface-base-box-shadow);
```

Comment:
`border-radius` and `box-shadow` per [design spec in Figma](https://dev-portal-git-zsone-image-in-the-darkness-bind-them-hashicorp.vercel.app/vault/docs/concepts/seal).